### PR TITLE
Added Rating Boxes and Made Style changes

### PR DESCRIPTION
**Pull request by celestinebuendia**
- Separated card into ClassCard component with its own props
- Added a right side Class Ratings section to the card, which displays the class ratings in boxes, or N/A if no ratings
- Added ratings CSS
- Slight CSS changes for card to better fit the Figma design
- Reordered and commented some of the CSS page for organization

Testing
- Saw changes in Vercel preview, on all directory pages

### CODE REVIEW

File: app/src/Homepage/Homepage.tsx [hunk 1]
```diff
@@ -157,6 +157,52 @@ export const HomePage: React.FC = () => {
     );
 }
 
+export const ClassRating: React.FC<{category: string, rating: number, type: string}> = ({category, rating, type}) => {
+  const dynamicclassname = `rating-pair-rating ratingbox-${type}`;
```
Comment (jeremynguyen593) [inline app/src/Homepage/Homepage.tsx:161]: Can you make this camelCased? I think that's what the style guide tells us

**Review by jeremynguyen593 — CHANGES_REQUESTED** [review]

File: app/src/Homepage/Homepage.tsx (hunk 1 above)
Comment (celestinebuendia) [reply app/src/Homepage/Homepage.tsx:161]: Sure. Actually just copied this from Thao's homepage code so I'll change her's too.

File: app/src/Homepage/Homepage.tsx [hunk 2]
```diff
@@ -152,11 +152,58 @@ export const HomePage: React.FC = () => {
                 seen the material in the industry?
               </RatingDesc>
             </div>
+            <ClassCard name='Computer Communication and Networks' num={561} desc='Fundamentals of data transmission: coding, message formats, and protocols. Organization of computer networks. Examples of network implementations. Prerequisite: either CSE 451 or equivalent.' rating1={2.5} rating2={3} rating3={5}/>
```
Comment (jeremynguyen593) [inline app/src/Homepage/Homepage.tsx:155]: Is there a reason why we're hardcoding this in?

**Review by jeremynguyen593 — CHANGES_REQUESTED** [review]

File: app/src/Homepage/Homepage.tsx (hunk 2 above)
Comment (celestinebuendia) [reply app/src/Homepage/Homepage.tsx:155]: OOPs haha no that was just a test card that I put on the homepage for easy CSS viewing -- forgot to remove

**Review by jeremynguyen593 — APPROVED** [review]

**Pull request merged**